webpack.config.js

```diff
@@ -26,7 +25,7 @@ JS_CONFIG = {
         filename: "assets/js/[name].min.js",
     },
     optimization: {
-        minimizer: [new UglifyJsPlugin()],
+        minimize: true,
```

Comment:
Hmm, but you are not actually using TerserPlugin for minimizing?

```Javascript
    optimization: {
        minimizer: [
            new TerserPlugin({
                terserOptions: {
                    compress: {
                        drop_console: false
                    },
                    keep_classnames: true,
                    keep_fnames: true
                }
            })
        ]
    },
```

Reply:
Webpack v5 comes with the latest terser-webpack-plugin out of the box. If you are using Webpack v5 or above and wish to customize the options, you will still need to install terser-webpack-plugin.

Reply:
I did not see any configuration of the UglifyPlugin. Are the options needed?